_pages/develop/map.md

```diff
@@ -0,0 +1,682 @@
+
+# Basic concepts
+
+Use the map to plot geographic data such as individual assets or clusters of
+assets and information about them. The data concepts underlying the map include:
+
+- **Base layer:** The base layer of a map renders tile images that include
+information such as streets, mountains, and national parks. For example,
+satellite, terrain, and map view are all different base layer styles.
+- **Overlay:** Information that is put on top of a map's base layer that is
+specific to the purpose of that map. For example, traffic data or bike lane
+information might be set as an overlay on a road map.
+- **Control:** A map layer that allows the user to manipulate the map, e.g. by
+zooming in, out, or to their current location.
+- **CRS:** A CRS is a coordinate reference system (also known as a coordinate
+relationship system or coordinate projection system) used to project
+geographical points into pixel coordinates.
+
+## Graphic: map layers
+
+# Subcomponents
+
+The following subcomponents are available for use with px-map:
+
+**Base map**
+
+- `<px-map>`: Draws the underlying map, sets location/zoom and notifies updates
+when the user interacts with the map, enables and disables interactions,
+orchestrates subcomponents
+
+**Tile layers**
+
+- `<px-map-tile-layer>`: Calls a tile service API to fetch underlying tile images
+for the map
+- `<px-map-tile-layer-bing>`: Calls the Bing Maps API to fetch underlying tile
+images for the map
+- `<px-map-tile-layer-google>`: Calls the Google Maps API to fetch underlying
+tile images for the map
+- `<px-map-tile-layer-wms>`: Displays WMS services as tile layers on the map.
+
+**Overlay/visualization layers**
+
+- `<px-map-layer-geojson>`: Draws GeoJSON data as vectors on the map
+- `<px-map-layer-group>`: Groups related overlays together to allow for bulk
+interactions (e.g. hide all in the group)
+- `<px-map-marker-static>`: Creates a marker that shows the state of a point/asset
+(e.g. with color)
+- `<px-map-marker-symbol>`: Creates a marker with an icon that shows the state
+of a point/asset
+- `<px-map-marker-locate>`: Creates a marker that represents the user's location
+- `<px-map-marker-group>`: Draws many markers in clusters, useful for visualizing
+thousands of points that can be dynamically updated
+
+**UI components**
+
+- `<px-map-popup-info>`: Binds a popup that can include text or an image to a marker
+- `<px-map-popup-data>`: Binds a popup that can include text and key/value data
+to a marker
+
+**Controls**
+
+- `<px-map-control-zoom>`: Adds zoom buttons the user can tap to zoom in or out
+of the map
+- `<px-map-control-scale>`: Adds a scale that shows the distance of an area on
+the map in miles/kilometers
+- `<px-map-control-locate>`: Adds a button the user can tap to locate themselves
+on the map and center the map on their location
+
+# Set up in your app
+
+## Choosing a CRS
+
+A CRS is framework for defining real-world locations that attempts to reconcile
```

Comment:
Missing article and I would explain the acronym:
`A coordinate reference system (CRS) is a framework...`